week7/answers.sql

```diff
@@ -1,3 +1,314 @@
 -- Create your tables, views, functions and procedures here!
 CREATE SCHEMA destruction;
 USE destruction;
+
```

Comment:
Just an overall comment that I would recommend adding some more `NOT NULL` keywords to column definitions, unless you can think of explicit use cases where you would want a NULL value.